lib/ability.rb

```diff
@@ -34,7 +34,7 @@ def initialize(user, resource, controller)
     end
 
     can :list, Facility if user.facilities.size > 0 and controller.is_a?(FacilitiesController)
-    can :read, Notification if user.operator? || user.administrator?
+    can :read, Notification if user.notifications.active.any?
```

Comment:
While I left the facility operator `:read` in, this puts in the behavior we discussed earlier: if a user has notifications, the user may read them.